gapic/src/main/com/google/gapid/GraphicsTraceView.java

```diff
@@ -82,8 +83,7 @@ public GraphicsTraceView(Composite parent, Models models, Widgets widgets) {
 
     setLayout(new GridLayout(1, false));
 
-    new ReplayDeviceSelector(this, models)
-        .setLayoutData(new GridData(SWT.FILL, SWT.TOP, true, false));
+    deviceDialog = new DeviceDialog(this, getShell(), models, widgets);
```

Comment:
This line will leave a blank space above the main tabs, because there's a concrete composite `deviceDialog` created under `this` composite -- which is `graphicsTraceView `. 
Had some test but haven't found a very good way to avoid it. Change this line to `deviceDialog = new DeviceDialog(tabs, getShell(), models, widgets);` and move it after `tabs` creation can solve the blank area issue, but there might be better solution.

Reply:
I've changed for DeviceDialog to not extend Composite anymore.

We somehow need a widget to call `scheduleIfNotDisposed(someWidget, ...)`, I now use DeviceDialog parent Composite (in practice, GraphicsTraceView) as a widget here. The idea is that as long as the GraphicsTraceView is up, we can open the dialog (if need be).

I'm not sure how clean that is, but it seems to work at least.

@pmuetschard what do you think?